nipype/pipeline/plugins/tests/test_base.py

```diff
@@ -39,3 +40,34 @@ def func(arg1):
 
 wf.run(plugin='MultiProc')
 """
+
+@pytest.mark.regression
+def test_remove_nodes(tmpdir):
```

Comment:
You can avoid coercing to `pathlib.Path` if you just use `tmp_path`:

```suggestion
def test_remove_nodes(tmp_path):
```